packages/core/src/app/payment/paymentMethod/StripeUPEPaymentMethod.tsx

```diff
@@ -124,19 +124,22 @@ const StripeUPEPaymentMethod: FunctionComponent<
 
 function mapFromCheckoutProps({ checkoutState }: CheckoutContextProps ) {
     const {
-        data: { getConfig, getCustomer },
+        data: { getConfig, getCustomer, getPaymentProviderCustomer },
     } = checkoutState;
     const config = getConfig();
     const customer = getCustomer();
+    const paymentProviderCustomer = getPaymentProviderCustomer();
 
     if (!config || !customer) {
         return null;
     }
 
+    const isStripeLinkAuthenticated = paymentProviderCustomer?.stripeLinkAuthenticationState;
```

Comment:
This line might throw an error after you pull last changes from master

Reply:
It is working fine too